src/test/java/org/folio/rest/impl/NoteTypesImplTest.java

```diff
@@ -448,7 +447,7 @@ public void shouldReturn404OnPutWhenNoteNotFound() throws IOException, URISyntax
   }
 
   @Test
-  public void shouldReturn400OnPutWhenRequestIsInvalid() {
+  public void shouldReturn422OnPutWhenRequestIsInvalid() {
```

Comment:
Just a quick sanity check here -> why did we decide on a 400 before and now a 422?

Reply:
@sduvvuri-ebsco just changed the name of the method to conform with response code